app/controllers/items_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ItemsController < ApplicationController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+    @items = @merchant.items
+    end
```

Comment:
solid! Now we can alos just use line 4 for the show and be able to call @items for that specific merchant great work!